### PR TITLE
Log errors from Slack API and flesh out README

### PR DESCRIPTION
**Pull request by duncanbeevers**
Flesh out README with some practical values and invocations for getting the client identifier.

Add a bit of logging to the app; specifically log Slack API errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+```
+# grep identifier -m 1 ~/Library/Application\ Support/Plexamp/config.json
+```
 - Make yourself a legacy slack token on [this page](https://api.slack.com/custom-integrations/legacy-tokens).
 
 Then run the app as follows:
 
 ```
-$ PAUSE_EMOJI=<paused-emoji> PLAY_EMOJI=<playing-emoji> TOKEN=<token> PLAYER=<player> node index.js
+$ PLAY_EMOJI="plexamp-dark" \
```
Comment (jsclayton) [inline README.md]: Consider using standard Slack emoji names in the README vs our internal ones.

File: index.js [hunk 2]
```diff
@@ -17,17 +17,33 @@ app.post('/', upload.single('thumb'), function (req, res, next) {
...
+      });
     }
   }
 
   res.sendStatus(200);
 });
 
-app.listen(process.env.PORT || 10000);
+const port = process.env.PORT || 10000;
+app.listen(port);
```
Comment (jsclayton) [inline index.js]: If we're going to wait for the app to be listening, this should be:

```JS
app.listen(port, () => {
  console.log('listening on port ' + port);
});
```

Comment (elan) [reply index.js]: I was gonna say, will the log ever fire as written? I thought `listen` was blocking.

File: index.js [hunk 3]
```diff
@@ -17,17 +17,33 @@ app.post('/', upload.single('thumb'), function (req, res, next) {
...
         headers: {'content-type' : 'application/x-www-form-urlencoded'},
         url:     'https://slack.com/api/users.profile.set',
         body:    `${common}&profile=%7B%22status_text%22%3A%22%22%2C%22status_emoji%22%3A%22%3A${process.env.PAUSE_EMOJI}%3A%22%7D`
-      }, (error, response, body) => {});
+      }, (error, response, body) => {
+        const parsedBody = !error && response.statusCode === 200 && JSON.parse(body);
+        const errorMessage = error || (!parsedBody.ok && parsedBody.error);
+
+        if (errorMessage) {
+          console.log('error posting stopped / paused status: ' + errorMessage);
```
Comment (elan) [inline index.js]: Maybe pull this common code out into a function? 😅

File: index.js (hunk 2 above)
Comment (duncanbeevers) [reply index.js]: Blocking! No way! The whole program would fail to run in that case. Gotta keep that event loop free.

**Comment by elan** [comment]
Thanks, @duncanbeevers!

> Blocking! No way! The whole program would fail to run in that case. Gotta keep that event loop free.

Yeah, sorry, brain failure 🙄 For some reason I thought if it wasn't blocking the app would exit. I clearly no nothing about Javascript yet.

**Pull request merged**